warrior/Framework/ClassUtils/WSelenium/browser_mgmt.py

```diff
@@ -480,20 +480,23 @@ def _make_ff(self, webdriver_remote_url, desired_capabilites, profile_dir,
                         profile_dir)
             else:
                 ff_capabilities = webdriver.DesiredCapabilities.FIREFOX
-                if ff_capabilities['marionette']:
+                try:
                     ff_capabilities['acceptInsecureCerts'] = True
                     ffbinary = FirefoxBinary(binary)
                     browser = webdriver.Firefox(firefox_binary=ffbinary,
                                                 firefox_profile=profile_dir,
                                                 executable_path=gecko_path)
-                else:
+                except:
```

Comment:
The try except block ignore the error and force a Firefox browser to spawn
without logging the potential error